CMakeLists.txt

```diff
@@ -48,6 +48,11 @@ set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 set(CMAKE_LIBRARY_OUTPUT_DIRECTORY "${CMAKE_INSTALL_PREFIX}/lib")
 set(CMAKE_PREFIX_PATH "${CMAKE_INSTALL_PREFIX}")
 
+# Relative RPATH setup.
+list(APPEND CMAKE_INSTALL_RPATH $ORIGIN/../lib))
```

Comment:
You might need to escape the `$` somehow so it doesn't get expanded as a make or shell variable.  In a Makefile, you generally need `'$$ORIGIN'` for that.  Not sure what cmake needs.

Reply:
Allegedly, CMake is okay with $ORIGIN as is, only ${} needs to be escaped. But unfortunately I am unable to test that right now.